src/EditorFeatures/Test/BraceHighlighting/AbstractBraceHighlightingTests.cs

```diff
@@ -42,6 +43,6 @@ public abstract class AbstractBraceHighlightingTests
             }
         }
 
-        protected abstract Task<TestWorkspace> CreateWorkspaceAsync(string markup);
+        protected abstract Task<TestWorkspace> CreateWorkspaceAsync(string markup, CSharpParseOptions options);
```

Comment:
Done. Changed to `ParseOptions` to make room for future VB usage of this base class.